### PR TITLE
20.10.1 rns

### PR DESCRIPTION
**Pull request by moishce**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.10.1.md [hunk 1]
```diff
@@ -0,0 +1,859 @@
+# Cortex XSOAR Content Release Notes for version 20.10.1 (159259)
+##### Published on 27 October 2020
+
+### New: Cisco Umbrella cloud security Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.10.1.md:4]:
```suggestion
### New: Cisco Umbrella cloud security Pack v1.0.0 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.10.1.md [hunk 2]
```diff
@@ -0,0 +1,859 @@
...
+#### Integrations
+##### Cisco Umbrella Cloud Security  
+Adds domains to Umbrella block list.
+
+
+---
+
+### New: Cisco WebEx Feed Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.10.1.md:12]:
```suggestion
### New: Cisco WebEx Feed Pack v1.0.0 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.10.1.md [hunk 3]
```diff
@@ -0,0 +1,859 @@
...
+
+### New: ExportToXLSX Pack v1.0.0
+#### Scripts
+##### ExportToXLSX  
+Exports context data to a Microsoft Excel Open XML Spreadsheet (XLSX) file.
+
+
+---
+
+### New: Graylog Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.10.1.md:28]:
```suggestion
### New: Graylog Pack v1.0.0 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.10.1.md [hunk 4]
```diff
@@ -0,0 +1,859 @@
...
+#### Integrations
+##### Graylog  
+Integration with Graylog to search for logs and events.
+
+
+---
+
+### New: Hatching Triage Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.10.1.md:36]:
```suggestion
### New: Hatching Triage Pack v1.0.0 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.10.1.md [hunk 5]
```diff
@@ -0,0 +1,859 @@
...
+
+### New: Majestic Million Feed Pack v1.0.0
+#### Integrations
+##### Majestic Million Feed  
+Free search and download of the top million websites.
+
+
+---
+
+### New: Synapse Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.10.1.md:52]:
```suggestion
### New: Synapse Pack v1.0.0 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.10.1.md [hunk 6]
```diff
@@ -0,0 +1,859 @@
...
+##### Bambenek Consulting Feed  
+Maintenance and stability enhancements.
+
+---
+
+### Base Pack v1.3.20
+#### Scripts
+##### DBotPreProcessTextData  
+Updated the script to Python 3.
+Updated the script to Python 3.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.10.1.md:157]:
```suggestion
```

File: content-repo/extra-docs/releases/20.10.1.md [hunk 7]
```diff
@@ -0,0 +1,859 @@
...
+
+##### UnEscapeIndicatorIPv6  
+Fixed an issue where the script did not work as intended.
+
+##### ParseEmailFiles  
+Fixed an issue where parsing failed due to incorrect email payload filtering.
+
+---
+
+### Common Types Pack v2.1.1
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.10.1.md:320]: missing release notes for common types pack

File: content-repo/extra-docs/releases/20.10.1.md [hunk 8]
```diff
@@ -0,0 +1,859 @@
...
+- Updated docker image to the latest version.
+- Maintenance and stability enhancements.
+
+##### IntegrationsCheck-Widget-IntegrationsErrorsInfo  
+- Maintenance and stability enhancements.
+- Updated docker image to the latest version.
+
+---
+
+### Lacework Pack v1.0.1
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.10.1.md:561]:
```suggestion
### Lacework Pack v1.0.1 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.10.1.md [hunk 9]
```diff
@@ -0,0 +1,859 @@
...
+#### Integrations
+##### Lacework  
+- Updated description of the Integration to be more accurate.
+- Changed wording from 'instance' to 'account' for consistency across Lacework projects.
+- Upgraded the Docker image to demisto/lacework:1.0.0.12410.
+
+---
+
+### Luminate Pack v1.0.1
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.10.1.md:570]:
```suggestion
### Luminate Pack v1.0.1 (Community Contributed)
```

File: content-repo/extra-docs/releases/20.10.1.md [hunk 10]
```diff
@@ -0,0 +1,859 @@
...
+#### Dashboards
+##### Incidents Overview  
+Added Active Incidents by Source widget.
+
+#### Layouts
+#### Reports
+##### New: Case Report  
+Investigation Summary Report from Case Management Pack.
+
+#### Script
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.10.1.md:210]:
```suggestion
#### Scripts
```

**Comment by ShirleyDenkberg** [comment]
@Itay4 Doc review completed

File: content-repo/extra-docs/releases/20.10.1.md [hunk 11]
```diff
@@ -0,0 +1,850 @@
...
+##### Amazon DynamoDB  
+- Documentation and metadata improvements.
+- Updated the Docker image to: *demisto/boto3py3:1.0.0.12514*.
+
+---
+
+### Analyst1 Pack v1.0.4 (Partner Supported)
+#### Integrations
+##### illuminate  
+Documentation and metadata improvements.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.10.1.md:100]: I think we can remove this entry as we have it below.

**Review by Itay4 — APPROVED** [review]

**Pull request merged**